src/registry/images/registry.json

```diff
@@ -0,0 +1,751 @@
+{
+  "chrome-headless": {
+    "deprecated": [
+
+    ],
+    "docs": {
+      "relationship_name": "headless",
+      "service_name": "headless",
+      "url": "\/configuration\/services\/headless-chrome.html"
+    },
+    "endpoint": "http",
+    "name": "Headless Chrome",
+    "runtime": false,
+    "supported": [
+      {
+        "major": 73,
+        "minor": null
```

Comment:
Are major and minor split in the Manifest files and thus in what we're likely to get in the future from an automated system?  Otherwise this doesn't seem that helpful to split, since the user doesn't care about that distinction.